packages/x-license/src/useLicenseVerifier/useLicenseVerifier.ts

```diff
@@ -47,15 +52,15 @@ export function useLicenseVerifier(
       return sharedLicenseStatuses[packageName]!.licenseVerifier;
     }
 
-    const acceptedScopes: LicenseScope[] = packageName.includes('premium')
-      ? ['premium']
-      : ['pro', 'premium'];
+    const acceptedScopes: readonly LicenseScope[] = extractAcceptedScopes(packageName);
```

Comment:
The mode is correctly typed, I don't think we need to explicitly type the returned value here :ok_hand: 

```suggestion
    const acceptedScopes = extractAcceptedScopes(packageName);
```

Same below